src/Spectre.Algorithms/Methods/Segmentation.cs

```diff
@@ -66,7 +66,7 @@ public DivikResult Divik(IDataset dataset, DivikOptions options)
 
 			var varargin = options.ToVarargin();
 			var tmp = _segmentation.divik(numberOfOutputArgs, dataset.GetRawIntensities(), coordinates, varargin);
-			var result = new DivikResult(tmp);
+			var result = new DivikResult(tmp[1]);
```

Comment:
And also name tmp could be changed to matlabResult, so that its name truely resemblance what it really is.